daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/SBuiltinTest.scala

```diff
@@ -1496,8 +1496,7 @@ object SBuiltinTest {
     if (xs.isEmpty) "(Nil @Int64)"
     else xs.mkString(s"(Cons @Int64 [", ", ", s"] (Nil @Int64))")
 
-  private val entryFields: ImmArray[Ref.Name] =
-    ImmArray(Ref.Name.assertFromString("key"), Ref.Name.assertFromString("value"))
+  private val entryFields = Struct.assertFromSeq(List(keyFieldName -> (()), valueFieldName -> (())))
```

Comment:
We've already defined exactly the same in `SValue.scala`. Can't we reuse that here?